src/main/java/zeenea/connector/property/PropertyDefinition.java

```diff
@@ -192,6 +209,7 @@ public static UrlPropertyDefinition url(String code) {
    * @param label the label of the property definition
    * @return a new UrlPropertyDefinition instance
    */
+  @Deprecated(since = "2.3.0", forRemoval = true)
```

Comment:
Aussi : 
Et là également : 
```suggestion
  
  @Deprecated(
      since =
          "Deprecated since version 2.3.0, use url(name) instead. Scheduled for removal in version 3.0.0.",
      forRemoval = true)
```